### PR TITLE
 Resolve issue #21 Remaining items are not updating when we complete …

### PR DESCRIPTION
**Pull request by rakheesingh**
This PR addresses my proposal of #21 Remaining items are not updating when we complete any task #21

```` TodoList.js
let activeItemcount = applyFilter(list, FILTER_ACTIVE).length;
``````
change count variable to activeItemcount and instead of sending only length of total item, send the length of completed items.

This small change will solve bug #21.

### CODE REVIEW

File: src/components/ui/TodoList.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ import Info from './Info';
...
 import Footer from './Footer';
 import FilteredList from './FilteredList';
-import {applyFilter, search} from '../../services/filter';
+import {applyFilter, search, FILTER_ACTIVE} from '../../services/filter';
 
 export default function TodoList(props) {
     const {list, filter, mode, query} = props.data;
     const {addNew, changeFilter, changeStatus, changeMode, setSearchQuery} = props.actions;
-    const count = list.length;
+    let activeItemcount = applyFilter(list, FILTER_ACTIVE).length;
```
Comment (kabirbaidhya) [inline src/components/ui/TodoList.js:11]:
```suggestion
    const activeItemCount = applyFilter(list, FILTER_ACTIVE).length;
```

File: src/components/ui/Footer.js [hunk 2]
```diff
@@ -3,15 +3,15 @@ import Filter from './Filter';
 import ButtonWrapper from './ButtonWrapper';
 
 export default function Footer(props) {
-    const {count, filter, changeFilter} = props;
+    const {activeItemcount, filter, changeFilter} = props;
```
Comment (kabirbaidhya) [inline src/components/ui/Footer.js:6]:
```suggestion
    const {activeItemCount, filter, changeFilter} = props;
```

**Review by kabirbaidhya — CHANGES_REQUESTED** [review]
@rakheesingh Thanks for the PR. Have made few suggestions.

**Comment by rakheesingh** [comment]
Hey @kabirbaidhya thanks for reviewing, I followed your suggestion and pushed changes according to that.

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**